synapse/lib/stormlib/json.py

```diff
@@ -116,24 +116,21 @@ def getObjLocals(self):
         }
 
     @s_stormtypes.stormfunc(readonly=True)
-    async def _jsonSave(self, item, indent=None):
-        indent = await s_stormtypes.toint(indent, noneok=True)
+    async def _jsonSave(self, item, indent=False):
+        indent = await s_stormtypes.tobool(indent)
 
         try:
             item = await s_stormtypes.toprim(item)
-            return json.dumps(item, indent=indent)
-        except Exception as e:
+        except Exception:
```

Comment:
`toprim` allows `bytes`, `types.AsyncGeneratorType`, and `types.GeneratorType` through, which may also not be json safe. Are we relying on `s_json.dumps()` to catch and raise those errors as `SynError` subclasses now?

Reply:
We probably don't even need the toprim() check since s_json.dumps() will always raise MustBeJsonSafe.

Reply:
We need the toprim() for storm objects but yes we should let s_json.dumps() raise the error if the toprim()'d item is not json safe.